packages/wonder-blocks-testing/src/index.js

```diff
@@ -28,3 +28,11 @@ export type {
 } from "./fetch/types.js";
 export {mockGqlFetch} from "./gql/mock-gql-fetch.js";
 export type {GqlFetchMockFn, GqlMock, GqlMockOperation} from "./gql/types.js";
+
+// Test harness framework
+export * from "./harness/types.js";
+export * as harnessAdapters from "./harness/adapters/adapters.js";
+export {makeHookHarness} from "./harness/make-hook-harness.js";
+export {makeTestHarness} from "./harness/make-test-harness.js";
+export {hookHarness} from "./harness/hook-harness.js";
+export {testHarness} from "./harness/test-harness.js";
```

Comment:
I didn't think it was worth namespacing this API and types at a nested level of the exports. If we start seeing clashes of types and such, we would have to reconsider, of course.